lib/prelude.nim

```diff
@@ -7,16 +7,19 @@
 #    distribution, for details about the copyright.
 #
 
-# This is an include file that simply imports common modules for your
-# convenience:
-#
-# .. code-block:: nim
-#   include prelude
-#
-# Same as:
-#
-# .. code-block:: nim
-#   import os, strutils, times, parseutils, parseopt, hashes, tables, sets
+## This is an include file that simply imports common modules for your
+## convenience:
+##
+## .. code-block:: nim
+##   include prelude
+##
+## Same as:
+##
+## .. code-block:: nim
+##   # c backend
+##   import os, strutils, times, parseutils, parseopt, hashes, tables, sets
+##   # js backend
+##   import os, strutils, times, parseutils, hashes, tables, sets
```

Comment:
* keep it as `when not defined(js): import parseopt`, makes it visually obvious how these 
differ.

pre-existing (so nothing to do with your PR) but I dislike this module for several reasons:
* the list is historical and arbitrary
* it's trivial for users to defined their own include and choose what they want in their own project
I'd be in favor of deprecating this module

Reply:
Sure it's arbitrary, I cannot come up with a prelude include that wouldn't be arbitrary. And while it's easy enough to "come up" with one's own, hardly anybody does and this one is more standard.

Reply:
@araq well i do, but anyways, if it's gonna be in stdlib it should be more verstatile rather than 1-size-fits-none.
how about this, much more useful IMO and flexible:
```nim
lib/prelude2.nim
template importCommonJs =
  import jsconsole, ...
template importCommonCollections =
  import tables, sets, hashes, ...
template importCommonMath =
  import sums, sets, ...
template importCommonRepl = # can call above
  importCommonCollections()
importCommonMath()
```
doesn't need a hierarchy, ie you can have overlapping categories as shown above

Reply:
Why not simply ``include prelude / math``, ``include prelude / collections`` then? But then why not import the collection that you actually used. 

Please read this 10 times:

**You don't get the convenience of "don't make me think" by providing options.**